autometa/common/external/download_dataset.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+COPYRIGHT
+Copyright 2021 Ian J. Miller, Evan R. Rees, Kyle Wolf, Siddharth Uppal,
+Shaurya Chanana, Izaak Miller, Jason C. Kwan
+
+This file is part of Autometa.
+
+Autometa is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Autometa is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with Autometa. If not, see <http://www.gnu.org/licenses/>.
+COPYRIGHT
+
+pulling data from google drive folder with simulated or synthetic communities
+"""
+
+
+import gdown
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def download_dataset(dataset, out_dirpath):
+    # provide list of database options as a dictionary with file_ids from google
+    simulated = {
+        "test": "1fy3M7RnS_HGSQVKidCy-rAwXuxldyOOv",
+        "78": "15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y",
+        "156": "13bkwFBIUhdWVWlAmVCimDODWF-7tRxgI",
+        "312": "1qyAu-m6NCNuVlDFFC10waOD28j15yfV-",
+        "625": "1FgMXSD50ggu0UJbZd1PM_AvLt-E7gJix",
+        "1250": "1KoxwxBAYcz8Xz9H2v17N9CHOZ-WXWS5m",
+        "2500": "1wKZytjC4zjTuhHdNUyAT6wVbuDDIwk2m",
+        "5000": "1IX6vLfBptPxhL44dLa6jePs-GRw2XJ3S",
+        "10000": "1ON2vxEWC5FHyyPqlfZ0znMgnQ1fTirqG",
+    }
+
+    # construct file id into a url to put into gdown
+    file_id = simulated[dataset]
```

Comment:
What happens if I use `download(dataset=9999, out_dirpath="$HOME/benchmarking_data"`? A check for the provided value should be put in place and a `KeyError` or `ValueError` raised if the provided value is incorrect.

```suggestion
    if dataset not in simulated:
        raise KeyError(f"dataset ({dataset}) not in simulated data. Choices: {simulated.keys()}")
    file_id = simulated[dataset]
```

Reply:
These days I'm using argparser, so this should be automatically handled to return an error.